### ISSUE TITLE
tkinter:When using light mode everything disappears: texts, images, buttons...

### ISSUE THREAD

**Issue by youssefhoummad** [marticliment/win32mica#13]
When using light mode everything disappears: texts, images, buttons...

![image](https://user-images.githubusercontent.com/25711246/215263987-c6931ea8-7140-4019-b367-3a72600370ff.png)

As for when using the dark mode, everything looks perfect

![image](https://user-images.githubusercontent.com/25711246/215264142-0c149c91-f3e5-4748-82ea-d2f0b0da2703.png)

python 3.11
tkinter
windows 11

**Comment by marticliment** [comment]
can you please share your code in a zip file so I can test it?
Thanks!

**Comment by youssefhoummad** [comment]
> can you please share your code in a zip file so I can test it? Thanks!


[tkinter mica.zip](https://github.com/marticliment/win32mica/files/10529461/tkinter.mica.zip)

**Comment by marticliment** [comment]
This approach works for me:
![image](https://user-images.githubusercontent.com/53119851/215318186-ccbec7c8-53dd-4234-831d-26dc750be6d9.png)

```python

from ctypes import windll
import tkinter as tk
from tkinter import ttk

import sv_ttk
import win32mica as mc

from src.button import RoundedButton






def _download_frame(root):
    download_frame = tk.Frame(root, bg=root['bg'], padx=20, pady=5, name="download_frame")
    image = tk.PhotoImage(file=r'img\logo-1.png')
    logo = tk.Label(download_frame, image=image, bg=root['bg'])
    logo.image = image
    logo.pack(pady=(20,20))

    tk.Label(root, bg=root['bg'], text="video downloader", font=16,fg='#D13438').pack()
    url_entry = ttk.Entry(download_frame, width=36, name="url_entry")
    url_entry.pack(ipady=3, pady=20)

    start_button = RoundedButton(download_frame, name="start_button", text="download video", width=298, bg="#D13438")
    start_button.pack( pady=(10,30))

    return download_frame




app=tk.Tk()

mode = mc.MICAMODE.DARK

if mode:
    app.configure(bg="#000000")
    app.wm_attributes("-transparent", "#000000")
    sv_ttk.use_dark_theme()
else:
    app.configure(bg="#ffffff")
    app.wm_attributes("-transparent", "#ffffff")
    sv_ttk.use_light_theme()
    

app.update()


HWND=windll.user32.GetParent(app.winfo_id())
mc.ApplyMica(HWND, ColorMode=mode) # here by using light mode everything disappears: texts, images, buttons..


home = _download_frame(app)
home.pack(fill='both', expand=True, anchor='nw')

app.mainloop()
```

**Comment by youssefhoummad** [comment]
by using `app.wm_attributes("-transparent", color)` make app unusable.

**Comment by marticliment** [comment]
Then this is the same issue as #11 and #7.

There's this comment that migtht help you: https://github.com/marticliment/win32mica/issues/11#issuecomment-1321114928

**Comment by youssefhoummad** [comment]
for dark mode no need `app.wm_attributes("-transparent", color)`, the effect mica is work very well.
so win32mica is fine.

so the problem in light mode, if i use `transparent color` i get effect mica but the app became entirely clickthrough, even titlebar.

thanks,

**Comment by marticliment** [comment]
Well, then the best thing I can say is to migrate to another UI toolkit.

I can personally recommend you Qt, because it provides lots of flexibility, performance with high quality.
I also reached a point where tkinter wasn't enough, and I decided to fully convert the project I was working on to Qt, and while it was a tedious process, I can guarantee that I have never regretted doing the change.

Tkinter gives poor handle of transparencies, and this issue is the result of those limted capabilities
